config/robotframework_aio/release_items_RobotFramework_UDS.json

```diff
@@ -109,10 +109,23 @@ for more details.
 
 * ``Write Data By Name``
 * ``Routine Control By Name``
+* ``Input Output Control By Name``
 * ``Write Data By Name`` features.
 * ``Get Encoded Request Message``
 * ``Get Decoded Response Message``
 
+Please refer to `RobotFramework_UDS documentation <https://github.com/test-fullautomation/robotframework-uds/blob/develop/RobotFramework_UDS/RobotFramework_UDS.pdf>`_
+for more details.
+"
+                                       ],
+                      "0.13.2." : [

```

Comment:
Hi Tri,
please change this to 0.13.1.
As I still wait for Son, I will merge it into 0.13.1.

Thank you,
 Thomas